clients/vault/src/oracle/constants.rs

```diff
@@ -16,7 +16,8 @@ pub const MAX_DISTANCE_FROM_CURRENT_SLOT: Slot = 3;
 pub const VALIDATOR_COUNT_TEST_NETWORK: usize = 3;
 pub const VALIDATOR_COUNT_PUBLIC_NETWORK: usize = 23;
 
-pub const MAX_SLOT_TO_REMEMBER: Slot = 12;
+/// increase the time window to 24 hours.
+pub const MAX_SLOTS_TO_REMEMBER: Slot = 14400;
```

Comment:
Nice. Let's maybe rewrite the comment though to give a bit more context.
```suggestion
/// Set the _expected_ `MAX_SLOTS_TO_REMEMBER` parameter to 14400 Slots on Stellar, 
/// ie. 24 hours. This parameter is only correct when connecting to SatoshiPay Stellar validators
/// as the configuration of these nodes deviates from the default configuration.
pub const MAX_SLOTS_TO_REMEMBER: Slot = 14400;
```

Reply:
https://github.com/pendulum-chain/spacewalk/pull/355/commits/d7e87245e8192f6599e87e334ce650a8187f1e01
done